language/move-prover/src/spec_translator.rs

```diff
@@ -98,18 +98,23 @@ pub enum FunctionEntryPoint {
     Indirect,
     /// Variant used for verification.
     Verification,
+    /// Mutated inlined variant without pre/post conditions. Used by
+    /// specification checks. Holds `rewritten_code_index` from `Spec`.
+    /// To distinguish between different mutated functions.
+    Mutated(usize),
```

Comment:
Do you have different mutated functions at one point? I thought to have seen there is only one.

In any case, I think we should not extend this enumeration. We should just rewrite the function and use existing entry point types.